llvm/docs/TableGen/ProgRef.rst

```diff
@@ -1815,6 +1816,10 @@ and non-0 as true.
   ``int``. If the result is not 0, the *then* expression is produced; otherwise
   the *else* expression is produced.
 
+``!initialized(``\ *a*\ ``)``
+  This is produces a 1 if *a* is not the unitialized value (``?``) and 0
+  otherwise.
+
```

Comment:
minor grammar tweak:
```suggestion
``!initialized(``\ *a*\ ``)``
  This operator produces 1 if *a* is not the uninitialized value (``?``) and 0
  otherwise.

```